assets/dev/js/frontend/handlers/base-carousel.js

```diff
@@ -294,7 +294,8 @@ export default class CarouselHandlerBase extends SwiperHandlerBase {
 	}
 
 	getSpaceBetween( device = null ) {
-		return elementorFrontend.utils.controls.getResponsiveControlValue( this.getElementSettings(), 'image_spacing_custom', 'size', device ) || 0;
+		const responsiveControlValue = elementorFrontend.utils.controls.getResponsiveControlValue( this.getElementSettings(), 'image_spacing_custom', 'size', device );
+		return Number( responsiveControlValue ) || 0;
```

Comment:
While testing different scenarios using the old spacing control, I noticed that the CSS variables could break the slider. I have added this Number() method to fix broken carousel widgets.

We could leave it out as well, as users won't be able to add new CSS variables anymore.